pkg/kubelet/volumes.go

```diff
@@ -134,6 +141,19 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 		if err != nil {
 			return nil, err
 		}
+		if hasFSGroup && builder.SupportsOwnershipManagement() {
+			path := builder.GetPath()
+
+			err = kl.chownRunner.Chown(builder.GetPath(), 0, int(fsGroup))
+			if err != nil {
+				glog.Errorf("Chown failed on %v: %v", path, err)
+			}
+
+			err = kl.chmodRunner.Chmod(builder.GetPath(), os.FileMode(0770|os.ModeSetgid))
```

Comment:
I think it should be 0770 for files -- don't you want to be able to write to files in the volume?